docs/anki-integration.md

```diff
@@ -118,3 +118,36 @@ Below are some troubleshooting tips you can try if you are unable to create new
 
 - If all of the buttons appear grayed out, then you should double-check your deck and model configuration settings.
 - If no icons appear at all, make sure that Anki is running in the background and that [AnkiConnect](https://foosoft.net/projects/anki-connect) has been installed.
+
+### Anki Deck Generation
```

Comment:
Is it always generating a deck? From what I can tell it simply adds notes

Reply:
It can be generating a deck or adding notes. Will try to make the naming better here.